### PR TITLE
[FEAT] Add the tutorial `getting started`

### PR DESCRIPTION
**Pull request by csouchet**
Closes https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/issues/24

**Live environment of examples**

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/add_tutorial_getting_started_link/examples/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,9 @@ especially when using bpmn-visualization prior version `0.27.0`.
 - [Hacktoberfest themes](demo/hacktoberfest-custom-themes/README.md) - special Hacktoberfest diagram with Hacktoberfest colors
 - [Monitoring of all process instances demo](demo/monitoring-all-process-instances/README.md) - show how to use `bpmn-visualization` to render the monitoring of all process instances for a defined process
 
+## Tutorial
```
Comment (tbouffard) [inline README.md:85]: ⚠️ should use plural for consistencies with other paragraphs

**Review by tbouffard — CHANGES_REQUESTED** [review]

File: tutorial/getting-started/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Getting started
+
+A Javascript getting started tutorial to `bpmn-visualization` using an online monitoring scenario.
+- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorial/getting-started/index.html)
+- to run locally, open the [index.html](index.html) directly in a Web Browser
+
+## ♻️ Usage
+To see the content of the iframe: https://process-analytics.github.io/bpmn-visualization-tutorial-getting-started.
+
+Its code is on the [solution](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/tree/solution) branch of [bpmn-visualization-tutorial-getting-started](https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started) repository.
```
Comment (tbouffard) [inline tutorial/getting-started/README.md:10]: **Proposal**

The code of the tutorial is hosted in a dedicated repository: https://github.com/process-analytics/bpmn-visualization-tutorial-getting-started/

This example displays the solution of the tutorial in an iframe. The solution itself is hosted at https://process-analytics.github.io/bpmn-visualization-tutorial-getting-started.

File: README.md [hunk 3]
```diff
@@ -82,6 +82,9 @@ especially when using bpmn-visualization prior version `0.27.0`.
...
+- [Quick start](tutorial/quick-start/README.md) - the simplest way to integrate `bpmn-visualization` in an HTML page
```
[the remark this answers is not in the document]
Comment (csouchet) [reply README.md:86]: Ok, so we should to rename 'demo' to "demos" too ?

File: tutorial/getting-started/README.md (hunk 2 above)
Comment (csouchet) [reply tutorial/getting-started/README.md:10]: LGTM 🙂

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tbouffard) [reply README.md:86]: arggggg. 😒

[the remark this answers is not in the document]
Comment (csouchet) [reply README.md:86]: I started the renaming of `tutorial` 😉

File: examples/index.html [hunk 4]
```diff
@@ -300,25 +301,47 @@ <h2 class="bg-dark">Demos</h2>
             </section>
 
             <section class="col-12 mt-2 mb-2">
-                <a id="bpmn-rendering" class="anchor-from-summary"></a>
-                <h2 class="bg-dark">BPMN Rendering</h2>
+                <a id="tutorials" class="anchor-from-summary"></a>
+                <h2 class="bg-dark">Tutorial</h2>
```
Comment (tbouffard) [inline examples/index.html:305]:
```suggestion
                <h2 class="bg-dark">Tutorials</h2>
```

File: examples/index.html [hunk 5]
```diff
@@ -300,25 +301,47 @@ <h2 class="bg-dark">Demos</h2>
...
 
+                    <div class="col-md-12 col-4 p-2">
+                        <a href="../tutorials/getting-started/index.html" class="card-link">
+                            <div class="card p-2">
+                                <div class="card-header">
+                                    <div class="card-title">Getting started</div>
+                                    <div class="card-subtitle"></div>
+                                </div>
+                                <div class="card-image">
+                                    <img src="static/img/preview/tutorial/getting-started.gif" class="img-responsive" alt="getting started">
```
Comment (tbouffard) [inline examples/index.html:331]: ⚠️ preview images should be stored in tutorial**s** directory as well

File: examples/index.html (hunk 4 above)
Comment (csouchet) [reply examples/index.html:305]: Good catch ^^

File: examples/static/js/link-to-sources.js [hunk 6]
```diff
@@ -14,11 +14,14 @@ if (window.location.hostname === 'cdn.statically.io') {
 else if (window.location.hostname === 'localhost' || window.location.protocol.startsWith('file')) {
     logLinkToGithub('Local environment detected')
     const pathname = window.location.pathname;
-    // we keep everything after the latest occurrence of /examples/ or /demo/
+    // we keep everything after the latest occurrence of /examples/, /demo/ or /tutorial/
```
Comment (tbouffard) [inline examples/static/js/link-to-sources.js:17]:
```suggestion
    // we keep everything after the latest occurrence of /examples/, /demo/ or /tutoriasl/
```

File: examples/static/js/link-to-sources.js [hunk 7]
```diff
@@ -14,11 +14,14 @@ if (window.location.hostname === 'cdn.statically.io') {
 else if (window.location.hostname === 'localhost' || window.location.protocol.startsWith('file')) {
     logLinkToGithub('Local environment detected')
     const pathname = window.location.pathname;
-    // we keep everything after the latest occurrence of /examples/ or /demo/
+    // we keep everything after the latest occurrence of /examples/, /demo/ or /tutoriasl/
```
Comment (tbouffard) [inline examples/static/js/link-to-sources.js:17]:
```suggestion
    // we keep everything after the latest occurrence of /examples/, /demo/ or /tutorials/
```

Comment (csouchet) [inline examples/static/js/link-to-sources.js:17]:
```suggestion
    // we keep everything after the latest occurrence of /examples/, /demo/ or /tutorials/
```

File: examples/static/js/link-to-sources.js [hunk 8]
```diff
@@ -14,11 +14,14 @@ if (window.location.hostname === 'cdn.statically.io') {
...
     let lastIndexOfInterestingElement = pathname.lastIndexOf('/examples/');
     if (lastIndexOfInterestingElement === -1) {
         lastIndexOfInterestingElement = pathname.lastIndexOf('/demo/');
     }
+    if (lastIndexOfInterestingElement === -1) {
+        lastIndexOfInterestingElement = pathname.lastIndexOf('/tutorial/');
```
Comment (tbouffard) [inline examples/static/js/link-to-sources.js:23]:
```suggestion
        lastIndexOfInterestingElement = pathname.lastIndexOf('/tutorials/');
```

Comment (csouchet) [inline examples/static/js/link-to-sources.js:23]:
```suggestion
        lastIndexOfInterestingElement = pathname.lastIndexOf('/tutorials/');
```

File: tutorials/getting-started/index.html [hunk 9]
```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2020 Bonitasoft S.A.
```
Comment (tbouffard) [inline tutorials/getting-started/index.html:2]:
```suggestion
Copyright 2022 Bonitasoft S.A.
```

File: tutorials/quick-start/README.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
-# Getting Started
+# Quick start
 
 Javascript example to demonstrate how to integrate `bpmn-visualization` in an html page.
-- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/display-bpmn-diagram/01-getting-started/index.html)
+- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorial/quick-start/index.html)
```
Comment (tbouffard) [inline tutorials/quick-start/README.md:4]:
```suggestion
- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorials/quick-start/index.html)
```

File: tutorials/getting-started/README.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
+# Getting started
+
+A Javascript getting started tutorial to `bpmn-visualization` using an online monitoring scenario.
+- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorial/getting-started/index.html)
```
Comment (tbouffard) [inline tutorials/getting-started/README.md:4]:
```suggestion
- [__⏩ live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/tutorials/getting-started/index.html)
```

**Review by tbouffard — APPROVED** [review]
✔️ tested with https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/fc01f649/examples/index.html

**Pull request merged**